Solutions/Endjin.SemVer.DotNetApi.Cli/Endjin/SemVer/DotNetApi/Cli/ApiCompareTool.cs

```diff
@@ -26,7 +26,7 @@ internal class ApiCompareTool : IHostedCommand
         private readonly ILogger<ApiCompareTool> logger;
 
         /// <summary>
-        /// Creates a <see cref="ApiCompareTool"/>.
+        /// Initializes a new instance of the <see cref="ApiCompareTool"/> class.
```

Comment:
Ugh. This is one of my pet peeves with StyleCop: there is nothing this more verbose wording tells you that wasn't conveyed by the much simpler text it replaced.

This is why `Endjin.RecommendedPractices` disables this particular warning by default. That didn't happen when you upgraded `Endjin.RecommendedPractices` because if stylecop settings are already present, it does nothing—it doesn't overwrite what's already there.

Normally when I upgrade `Endjin.RecommendedPractices`, I delete the `.editorconfig` and `StyleCop.ruleset` files, and then rebuild. It reinstates them using our current latest preferred defaults, and then you can look through the diff to see if it made any changes you really didn't want.

Reply:
...although if you do revert this you could change "a" to "an" while you're in there, because that was wrong.